.github/workflows/ci-isotovideo.yml

```diff
@@ -6,10 +6,13 @@ jobs:
   test:
     runs-on: ubuntu-latest
     name: "CI: Running isotovideo test"
+    container:
+      image: perldocker/perl-tester:5.32
     steps:
     - uses: actions/checkout@v3
     - name: Run isotovideo tests
-      run: make test-isotovideo
+      run: |
+        make test-isotovideo
```

Comment:
I guess apt-get install podman is the way to go here, however that this depends on podman/docker to run... it's insane, can you create a ticket for this?